boards/riscv32/hifive1/doc/hifive1.rst

```diff
@@ -0,0 +1,60 @@
+.. _hifive1:
+
+SiFive HiFive1
+##############
+
+Overview
+********
+
+The HiFive1 is a development board with the FE310 RISC-V SoC.
+The board is Arduino compatible.
+More information can be found on `SiFive's wesite <https://www.sifive.com/products/hifive1>`_.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``HiFive1`` board configuration can be built as usual (see :ref:`build_an_application`).
+In order to build the application for ``HiFive1``, set the ``BOARD`` variable to ``hifive1``.
+
+.. code-block:: bash
+
+	$ export ZEPHYR_GCC_VARIANT="zephyr"
```

Comment:
We're trying not to include the ``$`` prompt to make it easier to cut and paste commands.  Also, don't indent so far (use spaces rather than a tab):

    .. code-block:: bash

       export ZEPHYR_GCC_VARIANT="zephyr"
       export ZEPHYR_SDK_INSTALL_DIR="/opt/zephyr-sdk" #assuming default sdk install directory
       export BOARD="hifive1"

Actually, are the first two ``export`` commands needed?